### PR TITLE
fix:divider bugfix

### PR DESCRIPTION
**Pull request by Z-AOAO**
## 📝 Description

Add a brief description.

## 💣 Is this a breaking change (Yes/No):

- [ ] Yes
- [x] No

## 🚧 How to migrate?

Don't need.

## 📝 Additional Information

lol

### CODE REVIEW

**Review by zch-modier — APPROVED** [review]

File: src/page/App/components/PanelSetters/ColorPickerSetter/index.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        defaultColor={_value}
+        onColorChange={(value) => {
+          handleUpdateDsl(attrName, hsvaToHexa(value))
+        }}
+      />
+    </div>
+  )
+}
+
+ColorPickerSetter.displayName = "BaseInput"
```
Comment (DoubleWoodLin) [inline src/page/App/components/PanelSetters/ColorPickerSetter/index.tsx:29]: 这个是写错了吗

File: src/page/App/components/WidgetPickerEditor/components/ColorPicker/index.tsx [hunk 2]
```diff
@@ -112,6 +114,7 @@ function ColorPicker(props: ColorPickerProps) {
           }}
           onBlur={() => {
             setInputFocus(false)
+            props.onColorChange && props.onColorChange(hsva)
```
Comment (DoubleWoodLin) [inline src/page/App/components/WidgetPickerEditor/components/ColorPicker/index.tsx:117]: ?.

File: src/page/App/components/WidgetPickerEditor/components/ColorPicker/index.tsx [hunk 3]
```diff
@@ -148,6 +151,7 @@ function ColorPicker(props: ColorPickerProps) {
                 onBlur={() => {
                   setInputFocus(false)
                   setAlphaPercentValue(hsva.a * 100 + "%")
+                  props.onColorChange && props.onColorChange(hsva)
```
Comment (DoubleWoodLin) [inline src/page/App/components/WidgetPickerEditor/components/ColorPicker/index.tsx:154]: ?.

File: src/page/App/components/WidgetPickerEditor/components/ColorPicker/index.tsx [hunk 4]
```diff
@@ -42,12 +42,15 @@ function ColorPicker(props: ColorPickerProps) {
...
-    setAlphaPercentValue(updateAlphaInputValue(hsva.a))
-    setInputValue(hsvaToHex(hsva))
-    props.onColorChange && props.onColorChange(hsva)
-  }, [])
+  const handleColorPick = useCallback(
+    (hsva: HsvaColor) => {
+      setHsva(hsva)
+      setAlphaPercentValue(updateAlphaInputValue(hsva.a))
+      setInputValue(hsvaToHex(hsva))
+      props.onColorChange && props.onColorChange(hsva)
```
Comment (DoubleWoodLin) [inline src/page/App/components/WidgetPickerEditor/components/ColorPicker/index.tsx:50]: ?.

File: src/widgetLibrary/ButtonWidget/button.tsx [hunk 5]
```diff
@@ -50,8 +50,6 @@ export const WrappedButton = forwardRef<any, WrappedButtonProps>(
           buttonRadius={borderRadius}
           leftIcon={leftIcon}
           rightIcon={rightIcon}
-          // borderColor={_borderColor}
-          // backgroundColor={backgroundColor}
           // textColor={_textColor}
```
Comment (DoubleWoodLin) [inline src/widgetLibrary/ButtonWidget/button.tsx:53]: 这行要保留吗

File: src/widgetLibrary/TextWidget/style.tsx [hunk 6]
```diff
@@ -38,11 +56,15 @@ export function applyTextStyle(
 }
 
 export function applyLinkStyle(linkColor?: string): SerializedStyles {
+  const _linkColor =
+    linkColor && innerColor.indexOf(linkColor) > -1
```
Comment (DoubleWoodLin) [inline src/widgetLibrary/TextWidget/style.tsx:60]: ?.

File: src/widgetLibrary/TextWidget/style.tsx [hunk 7]
```diff
@@ -27,8 +41,12 @@ export function applyTextStyle(
   linkColor?: string,
   backgroundColor?: string,
 ): SerializedStyles {
+  const _color =
+    textColor && innerColor.indexOf(textColor) > -1
```
Comment (DoubleWoodLin) [inline src/widgetLibrary/TextWidget/style.tsx:45]: ?.

**Review by DoubleWoodLin — APPROVED** [review]

**Pull request merged**